#Expyriment 0.9.0 (Python 2.7.15), .xpe-file, coding: cp1252
#date: Mon Jul 16 2018 12:28:00
#sha1: a46e95
#modules: 
#display: size=(800, 600), window_mode=True, open_gl=2
#os: ('Windows', 'EDU-JBOOTH-MOB', '10', '10.0.17134', 'AMD64', 'Intel64 Family 6 Model 142 Stepping 9, GenuineIntel')
Time,Type,Event,Value,Detail,Detail2
11739,design,log,
#design: #exp: whole_number_to_position
#design: block_cnt,block_id,trial_cnt,trial_id
11739,design,logged,
11739,Experiment,started
11752,Stimulus,presented,1
11759,Stimulus,presented,1
11772,Stimulus,presented,1
11786,Stimulus,presented,2
12616,Stimulus,presented,2
12711,Stimulus,presented,2
12810,Stimulus,presented,2
13009,Stimulus,presented,2
13249,Stimulus,presented,2
14049,Stimulus,presented,2
14134,Stimulus,presented,2
14197,Stimulus,presented,2
14689,Stimulus,presented,2
14727,Stimulus,presented,2
15270,TextInput,entered,practice20
15283,Stimulus,presented,2
15289,Stimulus,presented,2
15306,Stimulus,presented,2
15319,Stimulus,presented,3
15934,Stimulus,presented,3
16026,Stimulus,presented,3
16581,TextInput,entered,p20
16583,stderr,received,'Traceback (most recent call last):'
16583,stderr,received,'  File "C:\\Users\\stemlab\\Documents\\GitHub\\tft\\whole_number_to_position.py", line 72, in <module>'
16584,stderr,received,'    num2pos.create_experiment_objects(exp_info)'
16585,stderr,received,'  File "C:\\Python27\\lib\\site-packages\\trajtrackerp\\num2pos\\_n2p_init.py", line 55, in create_experiment_objects'
16585,stderr,received,'    create_numberline(exp_info)'
16585,stderr,received,'  File "C:\\Python27\\lib\\site-packages\\trajtrackerp\\num2pos\\_n2p_init.py", line 92, in create_numberline'
16586,stderr,received,'    common.validate_config_param_type("max_numberline_value", Number, config.max_numberline_value)'
16586,stderr,received,'  File "C:\\Python27\\lib\\site-packages\\trajtrackerp\\common\\_common_funcs_init.py", line 569, in validate_config_param_type'
16587,stderr,received,'    raise ttrk.TypeError("config.{:} was set to a non-{:} value ({:})".format(param_name, type_name, param_value))'
16587,stderr,received,'trajtracker.TypeError: TypeError: config.max_numberline_value was set to a non-number value (10,000)'
